conda-adapter/src/test/java/com/artipie/conda/CondaSliceS3ITCase.java

```diff
@@ -0,0 +1,370 @@
+                                .add("accessKeyId", "minioadmin")
+                                .add("secretAccessKey", "minioadmin")
+                                .build()
+                        )
+                        .build()
+                )
+            );
+        this.server = new VertxSliceServer(
+            CondaSliceS3ITCase.VERTX,
+            new LoggingSlice(new BodyLoggingSlice(new CondaSlice(storage, url, events))),
+            this.port
+        );
+        this.server.start();
+        MatcherAssert.assertThat(
+            "Failed to start Minio",
+            this.exec("bash", "-c", "nohup /root/bin/minio server /var/minio 2>&1|tee /tmp/minio.log &"),
+            this.exitCodeStrMatcher(0)
+        );
+        MatcherAssert.assertThat(
+            "Failed to wait for Minio",
+            this.exec("timeout", "30",  "sh", "-c", "until nc -z localhost 9000; do sleep 0.1; done"),
+            this.exitCodeStrMatcher(0)
+        );
+        MatcherAssert.assertThat(
+            "Login was not successful",
+            this.exec("anaconda login --username any --password any".split(" ")),
+            this.exitCodeStrMatcher(0)
+        );
+    }
+
+    @AfterEach
+    void stop() {
+        this.server.stop();
+        this.cntn.stop();
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "noarch_glom-22.1.0.tar.bz2,glom/22.1.0/noarch,noarch",
+        "snappy-1.1.3-0.tar.bz2,snappy/1.1.3/linux-64,linux-64"
+    })
+    void canSingleUploadToArtipie(final String pkgname, final String pkgpath, final String pkgarch)
+        throws Exception {
+        MatcherAssert.assertThat(
+            "repodata.json must be absent in S3 before test",
+            this.exec(String.format("curl -f -kv http://localhost:9000/buck1/%s/repodata.json", pkgarch).split(" ")),
+            this.exitCodeStrMatcher(CondaSliceS3ITCase.CURL_NOT_FOUND)
+        );
+        MatcherAssert.assertThat(
+            String.format("%s must be absent in S3 before test", pkgname),
+            this.exec(String.format("curl -f -kv http://localhost:9000/buck1/%s/%s", pkgarch, pkgname).split(" ")),
+            this.exitCodeStrMatcher(CondaSliceS3ITCase.CURL_NOT_FOUND)
+        );
+        MatcherAssert.assertThat(
+            "Package was not uploaded successfully",
+            this.exec(String.format("timeout 30s anaconda --show-traceback --verbose upload /w/%s", pkgname).split(" ")),
+            new StringContainsInOrder(
+                new ListOf<>(
+                    String.format(CondaSliceS3ITCase.EXIT_CODE_FMT, 0),
+                    String.format("Using Anaconda API: http://host.testcontainers.internal:%d/", this.port),
+                    String.format("Uploading file \"anonymous/%s/%s\"", pkgpath, pkgname),
+                    "Upload complete"
+                )
+            )
+        );
+        MatcherAssert.assertThat(
+            "repodata.json must be absent in file storage since file storage must be unused",
```

Comment:
assertion description seems off, s3 storage should be used, right?

Reply:
Thanks, actually this check is redundant here, since adapter won't have access to `/var/` anyway, so I remove it.